framework/include/userobjects/Terminator.h

```diff
@@ -51,7 +51,7 @@ class Terminator : public GeneralUserObject
   /// handle output of the optional message
   void handleMessage();
 
-  const enum class FailMode { HARD, SOFT } _fail_mode;
+  const enum class FailMode { HARD, SOFT, NONE } _fail_mode;
   const enum class ErrorLevel { INFO, WARNING, ERROR, NONE } _error_level;
```

Comment:
these two enums have too similar of a name IMO. I think we need some detailed doxygen and/or change these enum names to further delineate what the different roles of these are